lib/elixir/lib/option_parser.ex

```diff
@@ -492,4 +556,34 @@ defmodule OptionParser do
       :unknown
     end
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error_pluralize = pluralize(total, "error", "errors")
```

Comment:
You can find the same pattern [here][1].

[1]: https://github.com/elixir-lang/elixir/blob/master/lib/ex_unit/lib/ex_unit/cli_formatter.ex#L201-L202